app/models/concerns/findings/issues.rb

```diff
@@ -19,6 +19,50 @@ module Findings::Issues
     representativeness: 2
   }
 
+  module ClassMethods
+    def probability_risk_previous review, weakness_template = nil
+      quantity = 0
+
+      if weakness_template
+        quantity       = 1
+        current_review = review
+
+        4.times do
+          current_review = current_review&.previous
+
+          if review && previous_weakness_by_template?(current_review, weakness_template)
+            quantity += 1
+          end
+        end
+
+        quantity = csv_base quantity, weakness_template, review if FINDING_REPEATABILITY_FILE.include? review.organization.prefix
+      end
+
+      quantity
+    end
+
+    def csv_base quantity, weakness_template, review
+      csv_options  = { headers: true }
+      file         = FINDING_REPEATABILITY_FILE[current.organization.prefix]
+      project_name = review.plan_item.project
+      suc_id       = project_name[/\((\d+)\)/, 1]
+
+      CSV.foreach(file, csv_options) do |row|
+        if row['id_ofinal'] == weakness_template.reference && suc_id && row['id_suc'] == suc_id
+          (1..4).each do |idx|
+            quantity += (row["count#{idx}"] == '1' && quantity <= 5) ? 1 : 0
+          end
+        end
+      end
+
+      quantity
+    end
+
+    def previous_weakness_by_template? review, weakness_template
+      Array(review&.weaknesses).map(&:weakness_template_id).include? weakness_template.id
```

Comment:
```suggestion
review.weaknesses.where(weakness_template_id: weakness_template).present?
```
no obstante podes chequear con un if que review exista